libs/ng-morph/ng/index.ts

```diff
@@ -1,6 +1,6 @@
-export * from './helpers/get-bootstrap-fn';
 export * from './helpers/get-main-module';
```

Comment:
let's export helpers as well, they can be useful for someone

Reply:
My comment is slightly out of scope of this PR,
But if we plan to make them more "public" (by providing them in index) 
I'd like to suggest optional naming adjustments for better clarity:

`push-to-array-property` -> `push-to-decorator-array-property`.
`push-to-object-argument` -> `push-to-object-literal-property`

These ideas are optional :)

Reply:
Exported and renamed to make it clear what they do